apps/frontend/src/libs/enums/button-size.enum.ts

```diff
@@ -0,0 +1,9 @@
+const BUTTON_SIZE = {
+	DEFAULT: "default",
+	SMALL: "small",
+} as const;
+
+type ButtonSize = (typeof BUTTON_SIZE)[keyof typeof BUTTON_SIZE];
```

Comment:
```suggestion
type ButtonSize = ValueOf<typeof BUTTON_SIZE>
```
Also, do not export a type from enums folder